strangler-infusereo/code/ruby-service.rb

```diff
@@ -0,0 +1,37 @@
+require 'sinatra'
+require 'json'
+require 'net/http'
+
+set :port, 9494
+
+$users=[{name: 'foo', id: 1}]
+
+$delegate_user_call=false
+
+get "/v1/user" do
+  content_type :json
+  $users.to_json
+end
+
+get "/v1/user/:id" do
+  content_type :json
+  if ($delegate_user_call)
+    # call golang endpoint and return it
+    url = URI.parse('http://localhost:8080/v1/users/' + params[:id].to_s)
+    req = Net::HTTP::Get.new(url.to_s)
```

Comment:
No need to cast to string, I don't believe. Would need to test with Sinatra, though.